File: Search Admin Key/search_admin_key.js
const axios = require('axios');

const serviceName = 'your-azure-search-service';
const apiVersion = '2020-06-30';
const adminKey = 'your_azure_search_admin_key_here';
const url = `https://${serviceName}.search.windows.net/indexes?api-version=${apiVersion}`;

const headers = {
    'Content-Type': 'application/json',
    'api-key': adminKey
};

axios.get(url, { headers })
    .then(response => {
        console.log(response.data);
    })
    .catch(error => {
        console.error(error);
    });

File: Search Admin Key/search_admin_key.py
import requests

service_name = "your-azure-search-service"
api_version = "2020-06-30"
admin_key = "your_azure_search_admin_key_here"
url = f"https://{service_name}.search.windows.net/indexes?api-version={api_version}"

headers = {"Content-Type": "application/json", "api-key": admin_key}

response = requests.get(url, headers=headers)
print(response.json())
